lib/modules/dosomething/dosomething_taxonomy/dosomething_taxonomy.module

```diff
@@ -117,14 +117,7 @@ function dosomething_taxonomy_preprocess_taxonomy_term(&$vars) {
  * @return array
  */
 function dosomething_taxonomy_get_campaigns($tid) {
-  $nids = array();
-  $view = views_get_view('campaigns_by_term');
-  $view->set_arguments(array($tid));
-  $view->execute();
-  foreach ($view->result as $record) {
-    $nids[] = $record->nid;
-  }
-  return $nids;
+  return taxonomy_select_nodes($tid, FALSE);
```

Comment:
Good point! Personally I'm not sure but I never noticed it when testing. Even if it does I think its a better tradeoff in comparison to what we have now